cmd/ocm/create/cluster/cmd.go

```diff
@@ -174,6 +176,18 @@ func init() {
 	arguments.SetQuestion(fs, "version", "OpenShift version:")
 	Cmd.RegisterFlagCompletionFunc("version", arguments.MakeCompleteFunc(getVersionOptions))
 
+	fs.StringVar(
+		&args.domainPrefix,
+		"domain-prefix",
+		"",
+		"An optional unique domain prefix of the cluster. This will be used when generating a "+
+			"sub-domain for your cluster on openshiftapps.com. It must be unique and consist of "+
```

Comment:
If we ever change the URL from openshiftapps.com to something else, it means every customer that is using an older version of the CLI would see incorrect help message. That said, I don't have a workaround/alternative other than maybe writing something vague w/o specifying the exact domain. 
Anyhow, not a strong objection from writing it, just raising the awareness.

Reply:
That's a good point; I basically went with the help message that was in the `.name`.

Reply:
I could drop the `openshiftapps.com` from the text all together.

Reply:
done